model/src/commonMain/kotlin/authorization/ProductPermission.kt

```diff
@@ -46,6 +46,9 @@ enum class ProductPermission {
     /** Permission to create a [Repository] for the [Product]. */
     CREATE_REPOSITORY,
 
+    /** Permission to trigger an ORT run for the [repositories][Repository] of the [Product]. */
+    TRIGGER_ORT_RUN,
```

Comment:
This new permission also needs to be added to the `WRITER` role in `ProductRole.kt`.